neurow/lib/neurow/application.ex

```diff
@@ -10,15 +10,34 @@ defmodule Neurow.Application do
   def start(_type, _args) do
     public_api_port = Application.fetch_env!(:neurow, :public_api_port)
     internal_api_port = Application.fetch_env!(:neurow, :internal_api_port)
-    Logger.warning("Current host #{node()}")
-
-    Logger.warning("Starting internal API on port #{internal_api_port}")
-
     {:ok, ssl_keyfile} = Application.fetch_env(:neurow, :ssl_keyfile)
     {:ok, ssl_certfile} = Application.fetch_env(:neurow, :ssl_certfile)
-
     {:ok, history_min_duration} = Application.fetch_env(:neurow, :history_min_duration)
 
+    cluster_topologies =
+      Application.get_env(:neurow, :cluster_topologies, cluster_topologies_from_env_variables())
+
+    start(%{
+      public_api_port: public_api_port,
+      internal_api_port: internal_api_port,
+      ssl_keyfile: ssl_keyfile,
+      ssl_certfile: ssl_certfile,
+      history_min_duration: history_min_duration,
+      cluster_topologies: cluster_topologies
+    })
+  end
+
+  def start(%{
```

Comment:
I just split the logic of fetching the start parameters and the actual start logic.